website/source/docs/enterprise/index.html.md

```diff
@@ -3,20 +3,24 @@ layout: "docs"
 page_title: "Nomad Enterprise"
 sidebar_current: "docs-enterprise"
 description: |-
-  Nomad Enterprise adds collaboration and governance capabilities to Nomad.
-  Features include Namespaces, Resource Quotas, and Sentinel Policies.
+  Nomad Enterprise adds operations, collaboration, and governance capabilities to Nomad.
+  Features include Namespaces, Resource Quotas, Sentinel Policies, and Advanced Autopilot.
 ---
 
 # Nomad Enterprise
 
-[Nomad Enterprise](https://www.hashicorp.com/products/nomad/) adds collaboration
-and governance capabilities to Nomad. With Nomad Enterprise, operators can enable
-multiple teams to safely use a single multi-region deployment, limit resource
-consumption across teams or projects, and enforce arbitrary fine-grained policies
-upon job submission. Nomad Enterprise features include:
+[Nomad Enterprise](https://www.hashicorp.com/products/nomad/) adds operations, 
+collaboration, and governance capabilities to Nomad. Namespaces allow multiple 
+teams to safely use a single multi-region deployment. With Resource Quotas, 
```

Comment:
safely use a shared Nomad cluster by giving each team an isolated view of the cluster.